RHEL/7/input/checks/templates/template_kernel_module_disabled

```diff
@@ -0,0 +1,86 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_kernel_modules_disabled.py.  DO NOT EDIT.  -->
+  <definition class="compliance"
+  id="kernel_module_KERNMODULE_disabled" version="1">
+    <metadata>
+      <title>Disable KERNMODULE Kernel Module</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
```

Comment:
Same ```kernel_module_disabled``` template would be applicable to Fedora systems, right? => Add ```multi_platform_fedora``` as expected ```<platform>``` too. And possibly move this template into / under the ```/shared/oval``` directory, so it can be re-used also for Fedora modules.